client/src/extension.ts

```diff
@@ -156,6 +158,18 @@ export async function activate (context: vscode.ExtensionContext, launchOptions:
     client.sendNotification(jobs.Request.apiAddJar, { uri })
   })
 
+  // watch for changes to the flix.toml file
+  tomlWatcher = vscode.workspace.createFileSystemWatcher('**/flix.toml')
```

Comment:
The `flix.toml` file can only occur in the root, so perhaps we don't need the ** right?

Reply:
I didn't think of this. I suppose it would be better to only watch the single file instead of watching for `flix.toml` recursively